### PR TITLE
Stop attractivity criteria

### PR DESCRIPTION
**Pull request by pbench**
Adds two new criteria for loki

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/journey_common.py [hunk 1]
```diff
@@ -513,7 +513,15 @@ def __init__(self, output_type_serializer):
...
-            type=OptionValue(['classic', 'robustness', 'occupancy']),
+            type=OptionValue(
+                [
+                    'classic',
+                    'robustness',
+                    'occupancy',
+                    'arrival_stop_attractivity',
+                    'departure_stop_attractivity',
+                ]
+            ),
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/journey_common.py:524]: Shouldn't we hide the param for now and make it beta?

**Review by pbougue — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/journey_common.py (hunk 1 above)
Comment (pbench) [reply source/jormungandr/jormungandr/interfaces/v1/journey_common.py:524]: The "criteria" param already exists, and I think it needs to be in the navitia api schema to for our frontends.
I don't know if we can hide some values of a param ?

Comment (pbougue) [reply source/jormungandr/jormungandr/interfaces/v1/journey_common.py:524]: I don't think we can hide only values.
We may just document that they are beta below?
I don't know any front that uses the param itself, but you must know better, I trust you if you think it's not a good idea to hide the whole param.

**Review by xlqian — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/journey_common.py (hunk 1 above)
Comment (pbench) [reply source/jormungandr/jormungandr/interfaces/v1/journey_common.py:524]: I documented that the `criteria` feature is in beta

**Review by pbougue — APPROVED** [review]

**Review by azime — APPROVED** [review]

**Pull request merged**